app/src/main/res/values/styles.xml

```diff
@@ -20,6 +20,7 @@
         <item name="alertDialogStyle">@style/DialogStyleNormal</item>
         <item name="alertDialogTheme">@style/DialogStyleNormal</item>
         <item name="android:windowEnableSplitTouch">false</item>
+        <item name="android:navigationBarColor">@android:color/transparent</item>
```

Comment:
Why is this necessary? (add a comment)

Reply:
It lessens the "blow" when transitioning since the bottom bar isn't switching from grey to white.